src/vex.js

```diff
@@ -30,14 +30,10 @@ Vex.MakeException = (name) => {
 };
 
 // Default runtime exception.
-class RuntimeError {
+class RuntimeError extends Error {
```

Comment:
You just forgot to revert it or it has some reason to stay?

Reply:
"It is considered good practice to only throw the Error object itself or an object using the Error object as base objects for user-defined exceptions. **The fundamental benefit of Error objects is that they automatically keep track of where they were built and originated.**"
It helps future debug (I consider that part of the infraesctructure) and does not impact on the external interface.

Reply:
This is good, let it stay.